docs/user-guide/get-started.md

```diff
@@ -38,12 +38,13 @@ For example, to lint SCSS you can extend the [SCSS community config](https://www
 npm install --save-dev stylelint stylelint-config-standard-scss
 ```
 
-2\. Create a `.stylelintrc.json` configuration file in the root of your project with the following content:
+2\. Create a `stylelint.config.mjs` configuration file in the root of your project with the following content:
```

Comment:
[suggestion] `.mjs` or `.cjs` depends on the `type` value in `package.json`, so `.js` may be a bit more accurate in many cases.

```suggestion
2\. Create a `stylelint.config.js` configuration file in the root of your project with the following content:
```

Or if you want to use `.mjs`, some notes may be necessary, e.g.

```
Note: this example assumes that your project works under ESM.
```

Although that seems a bit verbose.